custom/README.md

```diff
@@ -0,0 +1,75 @@
+# Custom TeamCity Agent Images
```

Comment:
@Valrravn Could you please wording here?

Reply:
edited the entire readme a bit